src/app/components/VisualImage.js

```diff
@@ -3,18 +3,21 @@ function VisualImage(imageSource) {
   // FIXME: If we should use only base64/jpg, delete one condition in if
   const reg = /^data:image\/(?:gif|png|jpeg|bmp|webp)(?:;charset=utf-8)?;base64,(?:[A-Za-z0-9]|[+/])+={0,2}/;
   if (reg.test(imageSource) || /.jpg/.test(imageSource)) {
-    // Creating div and img
-    const createDiv = document.createElement('div');
-    createDiv.className = 'visual-box';
+    // check if image div is existing
+    const isDivExisting = document.querySelector('.visual-box');
```

Comment:
I don't think you shold querySelect anything outside of the component. If (supposedly) it is to be reused, it should only relate to itself, and be inserted by another function if needed.

Reply:
Will be refactored. For now it is only hotfix to not generate another tags